story_types.py

```diff
@@ -78,6 +198,39 @@ def common_elements(self) -> List[str]:
         return self.characteristics.get("common_elements", [])
 
 
+@dataclass
+class SubGenre:
+    """Represents a specific sub-genre within a genre."""
+    name: str
+    archetypes: List[str] = field(default_factory=list)
+    plot: str = ""
+    examples: List[str] = field(default_factory=list)
+    
+    def __str__(self) -> str:
+        return f"{self.name}: {self.plot}"
+
+
+@dataclass
+class Genre:
```

Comment:
I don't like this all being in the story_types.py file.  genre, subgenre and architypes should each have their own python files